canvas_manage_course/templates/canvas_manage_course/base.html

```diff
@@ -1,25 +1,28 @@
+{% extends 'icommons_ui/base_lti.html' %}
+
 {% load static %}
 
-<!DOCTYPE html>
-<html lang="en">
-    <head>
-        <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <title>Manage Course</title>
+{% block head_meta %}
+    {{ block.super }}
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+{% endblock head_meta %}
+
+{% block title %}Manage Course{% endblock title %}
+
+{% block stylesheet %}
+    {{ block.super }}
+    <link href="{% static 'font-awesome-4.5.0/css/font-awesome.min.css' %}" rel="stylesheet">
+{#  todo: use super() #}
+{% endblock stylesheet %}
 
-        <link href="{% static 'bootstrap-3.3.6/dist/css/bootstrap.min.css' %}" rel="stylesheet">
-        <link href="{% static 'font-awesome-4.5.0/css/font-awesome.min.css' %}" rel="stylesheet">
-        {% block stylesheets %}{% endblock stylesheets %}
-    </head>
-    <body>
-        <div class="container">
-            {% block breadcrumb %}{% endblock breadcrumb %}
-            {% block content %}{% endblock content %}
-        </div>
+{% block content %}
+    <div class="container">
+        {#  todo: rename breadcrumb #}
+        {% block dashboard_breadcrumb %}{% endblock dashboard_breadcrumb %}
+        {#  todo: rename content #}
+        {% block tool_content %}{% endblock tool_content %}
+    </div>
+{% endblock content %}
 
-        <script src="{% static 'jquery-1.12.0/jquery-1.12.0.min.js' %}"></script>
-        <script src="{% static 'bootstrap-3.3.6/dist/js/bootstrap.min.js' %}"></script>
-        {% block javascript %}{% endblock javascript %}
```

Comment:
Because this now extends base_lti.html, the javascript block in base_lti.html is inherited, which already has jquery and bootstrap. Templates that in turn extend this one can still access that block to inherit or override it if needed.